cl/audio/factories.py

```diff
@@ -15,8 +15,53 @@ class Meta:
     case_name = Faker("case_name")
     sha1 = Faker("sha1")
     download_url = Faker("url")
-    local_path_mp3 = FileField(upload_to="/tmp/audio")
-    local_path_original_file = FileField(upload_to="/tmp/audio/")
+
+    @classmethod
+    def _create(cls, model_class, *args, **kwargs):
+        """Creates an instance of the model class without indexing."""
+        obj = model_class(*args, **kwargs)
+        # explicitly sets `index=False` to prevent it from being indexed in SOLR.
```

Comment:
Can you add instructions here for what we should do later when Solr is removed? E.g.: 

```suggestion
        # explicitly sets `index=False` to prevent it from being indexed in SOLR.
        # Once Solr is removed, we can just remove this method completely. (IS THIS TRUE?)
```

Reply:
> Once Solr is removed, we can just remove this method completely. (IS THIS TRUE?)

Correct. Once SOLR is removed, we'll need to update [save method](https://github.com/freelawproject/courtlistener/blob/a0aa1716f157905913473e98b4ef226f7f205ea1/cl/audio/models.py#L199-L218) and factory for this model. The index argument is exclusively used by SOLR for indexing purposes and will no longer be necessary.

Reply:
Great. Can we tweak the comment to that effect, please?

Reply:
Sure!